src/App.js

```diff
@@ -1,12 +1,28 @@
 import React, { Component } from 'react';
+import { Route, Switch } from 'react-router-dom';
+
+import './app.scss';
+
+import { Button } from 'carbon-components-react';
```

Comment:
The button and button component import should have been deleted. See https://www.carbondesignsystem.com/tutorial/react/step-1#add-routing